.github/workflows/python-package.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
Is this comment accurate? Specifically, is there any linting going on here?

Reply:
You are right. That comment was from the template I took for the yml file. I have replaced it with an actual description of what is done.